pkgs/development/compilers/sbcl/bootstrap.nix

```diff
@@ -2,31 +2,13 @@
 
 let
   options = rec {
-    aarch64-darwin = {
-      version = "2.1.2";
-      system = "arm64-darwin";
-      sha256 = "sha256-H0ALigXcWIypdA+fTf7jERscwbb7QIAfcoxCtGDh0RU=";
-    };
-    x86_64-darwin = {
-      version = "2.2.9";
-      system = "x86-64-darwin";
-      sha256 = "sha256-b1BLkoLIOELAYBYA9eBmMgm1OxMxJewzNP96C9ADfKY=";
-    };
-    x86_64-linux = {
-      version = "1.3.16";
-      system = "x86-64-linux";
-      sha256 = "0sq2dylwwyqfwkbdvcgqwz3vay9v895zpb0fyzsiwy31d1x9pr2s";
-    };
+    # If CLISP (or any other non-binary-distributed Lisp) can run on any of
+    # these systems, their entry should be removed from this list.
     i686-linux = {
       version = "1.2.7";
       system = "x86-linux";
       sha256 = "07f3bz4br280qvn85i088vpzj9wcz8wmwrf665ypqx181pz2ai3j";
     };
-    aarch64-linux = {
-      version = "1.3.16";
-      system = "arm64-linux";
-      sha256 = "0q1brz9c49xgdljzfx8rpxxnlwhadxkcy5kg0mcd9wnxygind1cl";
-    };
     armv7l-linux = {
       version = "1.2.14";
```

Comment:
I've been conservative in what I remove here but honestly I'd be curious to see if any of these odd platforms still compile at all. Is anybody even testing them?

Reply:
At least for FreeBSD, Nixpkgs support for the platform previously existed but is currently broken.  There are some attempts to restore this support and get it right this time. I expect CLISP to work there fine once we have general platform support.